src/main/java/duke/command/ListCommand.java

```diff
@@ -0,0 +1,21 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+public class ListCommand extends Command {
+    /**
+     * Executes a list command using the given task list, UI and file storage.
+     *
+     * @param tasks the task list supplied.
+     * @param ui the UI supplied.
+     * @param storage the file storage supplied.
+     */
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        StringBuilder myBuilder = new StringBuilder();
+        myBuilder.append("Here are the tasks in your list:\n");
+        myBuilder.append(tasks.toString());
```

Comment:
Great job using a string builder to increase efficiency instead of using '+' to build the string object. There is a lot  of code here about appending each task to form the message which could be part of a ui class. You may want to consider letting the ui printing every message and leaving the message building to the ui class.